Sources/WishKit/SwiftUI/CreateWishView.swift

```diff
@@ -96,27 +99,27 @@ struct CreateWishView: View {
                 if WishKit.config.emailField != .none {
                     VStack(spacing: 0) {
                         HStack {
-                            if WishKit.config.emailField == .optional {
-                                Text("Email (optional)")
-                                    .font(.caption2)
-                                    .padding([.leading, .trailing, .bottom], 5)
-                            }
-
-                            if WishKit.config.emailField == .required {
-                                Text("Email (required)")
-                                    .font(.caption2)
-                                    .padding([.leading, .trailing, .bottom], 5)
-                            }
-
+                            Text(WishKit.config.localization.emailAddress + (WishKit.config.emailField == .optional ? " (optional)" : ""))
```

Comment:
It won't, but this is already the case in the main repro, and since we are not using the optional case, I haven't added this to the localization list.